### ISSUE TITLE
Requesting new Provider: ZoomCare

### ISSUE THREAD

**Issue by joestump** [fastenhealth/fasten-onprem#334]
This is a popular PCP in the Pacific Northwest. I couldn't find docs on how to log in/share.

**Comment by AnalogJ** [comment]
Unfortunately I dont see anything about a FHIR API for zoom care. 
Their patient portal doesn't mention anything about an underlying EHR platform, so I think it's custom built.
They also don't show up in the CHPL certified software platforms list - https://chpl.healthit.gov/#/search

https://help.zoomcare.com/hc/en-us/articles/360053033992-How-can-I-view-or-download-my-medical-records-

**Comment by joestump** [comment]
Yeah, I feared that. I believe they were recently bought by Providence Health. Perhaps they’ll migrate records.
